packages/custom-webpack/src/custom-webpack-schema.ts

```diff
@@ -1,9 +1,9 @@
 import {CustomWebpackBuilderConfig} from "./custom-webpack-builder-config";
+import {NormalizedBrowserBuilderSchema} from "@angular-devkit/build-angular";
+
 
 export interface CustomWebpackSchema {
 	customWebpackConfig: CustomWebpackBuilderConfig;
 }
 
-
-
-
+export interface NormalizedCustomWebpackBrowserBuildSchema extends NormalizedBrowserBuilderSchema, CustomWebpackSchema {}
```

Comment:
Placing it here breaks the abstraction (CustomWebpackSchema is common to every module in custom webpack but BrowserSchema is not).
Probably the best place to place it is in browser/schema.d.ts and then import it from there (it's fine that Karma module depends on Browser module)